deploy/osd-project-request-template/04-patch.label-default-namespace.yaml

```diff
@@ -1,7 +0,0 @@
-kind: Namespace
-apiVersion: v1
-name: default
-applyMode: AlwaysApply
-patch: |-
-  { "metadata": {"labels": {"name":"default"} } }
```

Comment:
Can't take this out without knowing who is using it.  Probably added to support NetworkPolicy for an internal customer.